client/CMakeLists.txt

```diff
@@ -145,6 +145,22 @@ set(client_HEADERS
 		SDLRWwrapper.h
 )
 
+if(APPLE_IOS)
+	set(client_SRCS ${client_SRCS}
+		CFocusableHelper.cpp
```

Comment:
Is it ios specific file?

Reply:
Yes, other platforms don't need this code.